core/stores/redis/redis.go

```diff
@@ -464,14 +464,14 @@ func (s *Redis) GetBit(key string, offset int64) (val int, err error) {
 }
 
 // Hdel is the implementation of redis hdel command.
-func (s *Redis) Hdel(key, field string) (val bool, err error) {
+func (s *Redis) Hdel(key string, field ...string) (val bool, err error) {
```

Comment:
multiple fields better to use fields instead of field.

Reply:
ok. Hdel field -> fields